examples/technology_showcase/example_05_td_026.py

```diff
@@ -113,6 +116,7 @@
 #materials.Import(mat_part_file_path)
 
 # Section 2 Set up the Unit System.
+ExtAPI.Application.ScriptByName("jscript").CallJScript("doGraphicsFit")
```

Comment:
@vnamdeo - Please avoid calling on to the jscript api which will not work in the future. You can use the Graphics API to do the same.

camera = Graphics.Camera
camera.SetFit()

please refer the Script documentation for more details.